ApiBundle/Controller/NodeController.php

```diff
@@ -80,6 +89,23 @@ public function showOrCreateAction(Request $request, $nodeId)
     }
 
     /**
+     * @param Request $request
+     * @param string $nodeId
+     *
+     * @Config\Route("/{nodeId}/show-or-create-error", name="open_orchestra_api_node_show_or_create_error")
+     * @Config\Method({"GET"})
+     * @Api\Serialize()
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_TREE_NODE')")
+     *
+     * @return FacadeInterface
+     */
+    public function showOrCreateErrorAction(Request $request, $nodeId)
+    {
+        return $this->showOrCreateAction($request, $nodeId, true);
```

Comment:
This method is quite useless, you could directly put the route on the other method and adding a default value for the `$errorNode`